components/raftstore/src/store/async_io/write_tests.rs

```diff
@@ -645,3 +680,88 @@ fn test_basic_flow_with_states() {
 
     t.writers.shutdown();
 }
+
+#[test]
+fn test_resource_group() {
+    let region_1 = 1;
+    let region_2 = 2;
+
+    let resource_manager = Arc::new(ResourceGroupManager::default());
+    let get_group = |name: &str, read_tokens: u64, write_tokens: u64| -> ResourceGroup {
+        let mut group = ResourceGroup::new();
+        group.set_name(name.to_string());
+        group.set_mode(GroupMode::RawMode);
```

Comment:
Please also add a case for RU mode.

Reply:
For different RU mode, they are covered in resource manager tests. For this PR, it is just to init the resource group meta and RU mode is not critical here.